examples/apps/kme/workload/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 Intel Corporation
```

Comment:
KME workload is part of core framework. Keep this in common/sgx_workload directory

Reply:
I have included kme workload as part of the enclave.